src/pens.rs

```diff
@@ -1,73 +1,27 @@
 //! Our own transformed bezier pen to avoid a dependency on write-fonts which is not in google3
 
-use kurbo::{Affine, BezPath, PathEl, Point};
+use kurbo::{Affine, BezPath, Point};
 use skrifa::outline::OutlinePen;
-use std::fmt::Write;
 
 /// Produces an svg representation of a font glyph corrected to be Y-down (as in svg) instead of Y-up (as in fonts)
 pub(crate) struct SvgPathPen {
-    transform: Affine,
     path: BezPath,
 }
 
-fn round2(v: f64) -> f64 {
-    (v * 100.0).round() / 100.0
-}
-
-fn push_point(svg: &mut String, prefix: char, p: Point) {
-    svg.push(prefix);
-    write!(svg, "{},{}", round2(p.x), round2(p.y)).expect("We can't write into a String?!");
-}
-
 impl SvgPathPen {
     pub(crate) fn new() -> Self {
         Self {
-            transform: Affine::FLIP_Y, // svg is Y-down, fonts are Y-up
             path: Default::default(),
         }
     }
 
     fn to_svg_units(&self, x: f32, y: f32) -> Point {
-        self.transform * Point::new(x as f64, y as f64)
+        // svg is Y-down, fonts are Y-up
+        Affine::FLIP_Y * Point::new(x as f64, y as f64)
```

Comment:
If we’re not doing arbitrary transforms, this should be equivalent to just negating the y coord.

Reply:
We might add back more than flip but for now it can just be -y indeed. Good catch.